services/ui/ws/server_window.cc

```diff
@@ -177,7 +177,28 @@ void ServerWindow::SetBounds(
     const base::Optional<cc::LocalSurfaceId>& local_surface_id) {
   if (bounds_ == bounds && current_local_surface_id_ == local_surface_id)
     return;
+  SetBoundsInternal(bounds, local_surface_id);
```

Comment:
in one place (ws::Display we are using SetBoundsImpl and here SetBoundsInternal)

"Internal" suffix it best, IMO.